diff --git a/lib/swc/default-options.js b/lib/swc/default-options.js
index 640985ff19bd2555dd40de74bee521851b2dd311..cc34b82ff8bc5d2c9bd1f3fc787fceb187e9b9c4 100644
--- a/lib/swc/default-options.js
+++ b/lib/swc/default-options.js
@@ -2,32 +2,34 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.defaultSwcOptionsFactory = void 0;
 const defaultSwcOptionsFactory = (tsOptions, configuration, extras) => {
-    return {
-        sourceMaps: tsOptions?.compilerOptions?.sourceMap ||
-            (tsOptions?.inlineSourceMap && "inline"),
-        module: {
-            type: "commonjs",
-        },
-        exclude: ["node_modules", "dist"],
-        jsc: {
-            target: "es2021",
-            parser: {
-                syntax: "typescript",
-                tsx: true,
-                decorators: true,
-                dynamicImport: true,
-            },
-            transform: {
-                legacyDecorator: true,
-                decoratorMetadata: true,
-                useDefineForClassFields: false,
-            },
-            keepClassNames: true,
-            baseUrl: tsOptions?.compilerOptions?.baseUrl,
-            paths: tsOptions?.compilerOptions?.paths,
-        },
-        minify: false,
-        swcrc: true,
-    };
+	return {
+		sourceMaps:
+			tsOptions?.compilerOptions?.sourceMap ||
+			(tsOptions?.inlineSourceMap && "inline"),
+		module: {
+			type: "es6",
+			resolveFully: true,
+		},
+		exclude: ["node_modules", "dist"],
+		jsc: {
+			target: "es2021",
+			parser: {
+				syntax: "typescript",
+				tsx: true,
+				decorators: true,
+				dynamicImport: true,
+			},
+			transform: {
+				legacyDecorator: true,
+				decoratorMetadata: true,
+				useDefineForClassFields: false,
+			},
+			keepClassNames: true,
+			baseUrl: tsOptions?.compilerOptions?.baseUrl,
+			paths: tsOptions?.compilerOptions?.paths,
+		},
+		minify: false,
+		swcrc: true,
+	};
 };
 exports.defaultSwcOptionsFactory = defaultSwcOptionsFactory;
